tests/SeleniumTests/__init__.py

```diff
@@ -39,6 +42,20 @@ def test_visit_basic_auth_secured_page(self):
         page_message = driver.find_element_by_css_selector('.example p').text
         self.assertTrue(page_message == 'Congratulations! You must have the proper credentials.')
 
+    def test_play_video(self):
+        driver = self.driver
+        driver.get('https://hls-js.netlify.com/demo/')
```

Comment:
Should we propose an PR to [the-internet](https://github.com/tourdedave/the-internet) since Netlify isn't under the umbrella of the Selenium community?

Perhaps we could add a Video section [here](http://the-internet.herokuapp.com/).\

Thoughts?

Reply:
I think we can tackle this in a different PR, and not as a prerequisite for this.
In general, we should try to come up with our own site that can be used for our tests and also for our docs.
I'll resolve this conversation, I hope that's ok.